src/exercise/01.md

```diff
@@ -68,4 +68,4 @@ using JavaScript as well.
 ## 🦉 Feedback
```

Comment:
According to my local these email changes should already be on `main` - not sure why this is coming in and I don't have time to fiddle with it right now.